src/ButtonStyles.js

```diff
@@ -4,83 +4,32 @@ import {get} from './constants'
 export default css`
   position: relative;
   display: inline-block;
-  padding: 6px 12px;
-  color: ${get('colors.gray.9')};
-  background-color: ${get('colors.gray.1')};
-  background-image: linear-gradient(-180deg, ${get('colors.gray.0')} 0%, ${get('colors.button.bg2')} 90%);
-  font-size: ${get('fontSizes.1')};
+  padding: 6px 16px;
   font-weight: ${get('fontWeights.bold')};
   line-height: 20px;
   white-space: nowrap;
   vertical-align: middle;
   cursor: pointer;
   user-select: none;
-  background-repeat: repeat-x;
-  background-position: -1px -1px;
-  background-size: 110% 110%;
-  border: 1px solid ${get('colors.button.border')};
-  border-radius: ${get('radii.1')};
+  border-radius: ${get('radii.2')};
   appearance: none;
   text-decoration: none;
 
   &:hover {
-    background-color: ${get('colors.button.hoverBg')};
-    background-image: linear-gradient(-180deg, ${get('colors.button.bg2')} 0%, ${get('colors.button.hoverBg')} 90%);
-    background-position: -0.5em center;
-    border-color: ${get('colors.blackfade35')};
+    // needed to override link styles
     text-decoration: none;
-    background-repeat: repeat-x;
-  }
-
-  &:active {
-    background-color: ${get('colors.button.activeBg')};
-    background-image: none;
-    box-shadow: ${get('colors.blackfade15')} 0px 0.15em 0.3em inset;
-    border-color: ${get('colors.button.border')}; //convert black to rbg here
   }
 
   &:focus {
     outline: none;
-    box-shadow: ${get('colors.button.focusShadow')} 0px 0px 0px 0.2em;
   }
 
-  &.grouped {
```

Comment:
All these styles are handled in the `ButtonGroup` component and the `group` class is not in use anymore.